docs/docs/configuration/configuring-superset.mdx

```diff
@@ -321,7 +321,103 @@ CUSTOM_SECURITY_MANAGER = CustomSsoSecurityManager
     }
   ]
   ```
+### Keycloak-Specific Configuration using Flask-OIDC
+If you are using Keycloak as OpenID Connect 1.0 Provider, the above configuration based on [`Authlib`](https://authlib.org/) might not work. In this case using [`Flask-OIDC`](https://https://pypi.org/project/flask-oidc/) is a viable option.
 
+Make sure the pip package [`Flask-OIDC`](https://https://pypi.org/project/flask-oidc/) is installed on the webserver. This was succesfully tested using version 2.2.0.
+
+For this to work, you need to create a new security manager in the same directory as `superset-config.py`, e.g. `keycloak_security_manager.py`:
```

Comment:
An overall question: will this work for people installing with different methods (Kubernetes, docker compose, from PyPI)?  Just thinking that this introduces two new files to be ingested alongside `superset_config.py`.  As long as that loading of config files happens based on directory and not particular file names, I think we're fine, but I wanted to confirm this method works for all deployment methods.

Reply:
Docker compose certainly works, because that's what I'm using. Don't know about K8s and PyPi.